Storage/ThirdParty/SwiftData.swift

```diff
@@ -218,6 +259,14 @@ open class SwiftData {
             }
         }
     }
+
+    public func cancel() {
+        sharedConnectionQueue.async {
```

Comment:
This is wrong. I need concurrent queue (non-serial) behaviour, neither sync nor async will get me that.